src/Polly.Core/Simmy/Behavior/BehaviorChaosStrategy.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Polly.Telemetry;
+
+namespace Polly.Simmy.Behavior;
+
+internal sealed class BehaviorChaosStrategy : MonkeyStrategy
+{
+    private readonly ResilienceStrategyTelemetry _telemetry;
+
+    public BehaviorChaosStrategy(
+        BehaviorStrategyOptions options,
+        ResilienceStrategyTelemetry telemetry)
+        : base(options)
+    {
+        Guard.NotNull(telemetry);
```

Comment:
No need to validate this because this code is internal and we have nullability warnings enabled.